### PR TITLE
Slim 762 il push alarm not

### PR DESCRIPTION
**Pull request by hannek24**

### CODE REVIEW

File: cucumber-tests-platform-dlms/src/test/java/com/alliander/osgp/platform/dlms/cucumber/steps/ws/smartmetering/smartmeteringconfiguration/PushNotificationAlarm.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2016 Smart Society Services B.V.
```
Comment (rudidevries) [inline cucumber-tests-platform-dlms/src/test/java/com/alliander/osgp/platform/dlms/cucumber/steps/ws/smartmetering/smartmeteringconfiguration/PushNotificationAlarm.java]: 2017

**Comment by villgust** [comment]
retest this please

**Comment by villgust** [comment]
retest this please

**Comment by villgust** [comment]
retest this please

**Pull request merged**